Source/Diagnostics/BoundaryScrapingDiagnostics.H

```diff
@@ -0,0 +1,56 @@
+/* Copyright 2022 Remi Lehe
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef WARPX_BOUNDARYSCRAPINGDIAGNOSTICS_H_
+#define WARPX_BOUNDARYSCRAPINGDIAGNOSTICS_H_
+
+#include "Diagnostics.H"
+
+#include <string>
+
+class
+BoundaryScrapingDiagnostics final : public Diagnostics
+{
+public:
+
+    BoundaryScrapingDiagnostics (int i, std::string name);
```

Comment:
TODO: the `...` parts
```suggestion
    /** Constructor
     *
     * @param i ...
     * @param name ...
     */
    BoundaryScrapingDiagnostics (int i, std::string name);
```